src/screens/PinScreen.js

```diff
@@ -98,10 +98,37 @@ class PinScreen extends React.Component {
     });
   }
 
+  /**
+   * Handle data migration on unlock screen
+   * This method is executed when the wallet is unlocked, so we can check
+   * if we need to change anything on the app data after an update
+   *
+   * @param {String} pin Unlock PIN written by the user
+   */
+  handleDataMigration = (pin) => {
```

Comment:
I'm not this is the best place to run the migrations, but it is better than `LoadHistoryScreen`.

Reply:
I think we should have a screen for data migration with a loading and an explanation for the user (and asking for the PIN, if needed). But this is just the fix for the current version, I don't think we should do this now, only when doing the data migration task

Reply:
I agree. We can move forward.